build.gradle

```diff
@@ -3,47 +3,65 @@ plugins {
+
+    gradleLint.rules = ['unused-dependency']
+
     repositories {
         jcenter()
         maven { url "https://jitpack.io" }
-        // Repository for running Embedded DynamoDB locally.
-        maven { url "https://s3.eu-central-1.amazonaws.com/dynamodb-local-frankfurt/release" }
     }
-}
-
-subprojects {
-    apply plugin: 'java-library'
-    apply plugin: 'checkstyle'
-    apply plugin: 'pmd'
-
 
     sourceCompatibility = JavaVersion.VERSION_11
     // source-code version and must be <= targetCompatibility
     targetCompatibility = JavaVersion.VERSION_11 // bytecode target version
 
     project.ext {
-        jacksonVersion = '2.10.2'
-        awsSdkVersion = '1.11.897'
+        nvaUserAccessServiceVersion = '0.3.1'
+        awsSdkVersion = '1.11.996'
         junit5Version = '5.6.0'
-        nvaCommonsVersion = '0.7.4'
-        nvaDatamodelJavaVersion = '0.11.5'
+        jacksonVersion = '2.12.3'
     }
 
     dependencies {
-        testImplementation group: 'com.github.BIBSYSDEV', name: 'nva-testutils', version: '0.1.16'
+        // not creating a project.ext variable to check if dependabot is actually picking this up.
+        implementation group: 'com.github.bibsysdev', name: 'core', version: '1.5.8'
+        implementation group: 'com.github.bibsysdev', name: 'secrets', version: '1.5.8'
+        implementation group: 'com.github.bibsysdev', name: 'apigateway', version: '1.5.8'
+
+        implementation group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: project.ext.jacksonVersion
+        implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: project.ext.jacksonVersion
+        implementation group: 'com.fasterxml.jackson.core', name: 'jackson-annotations', version: project.ext.jacksonVersion
+        implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: project.ext.jacksonVersion
+        implementation group: 'org.slf4j', name: 'slf4j-api', version: '1.8.0-beta4'
+        implementation group: 'com.amazonaws', name: 'aws-lambda-java-core', version: '1.2.1'
+        implementation group: 'com.amazonaws', name: 'aws-java-sdk-lambda', version: project.ext.awsSdkVersion
+        implementation group: 'com.amazonaws', name: 'aws-java-sdk-cognitoidp', version: project.ext.awsSdkVersion
+        implementation group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.9'
+        implementation group: 'org.zalando', name: 'problem', version: '0.25.0'
+
+        implementation group: 'com.github.BIBSYSDEV.nva-user-access-service', name: 'user-access-public-model', version: project.ext.nvaUserAccessServiceVersion
+        implementation group: 'com.github.BIBSYSDEV.nva-user-access-service', name: 'user-access-commons', version: project.ext.nvaUserAccessServiceVersion
+        implementation group: 'com.github.BIBSYSDEV.nva-user-access-service', name: 'user-access-errors', version: project.ext.nvaUserAccessServiceVersion
```

Comment:
Suggstion: add a TODO as reminder that these dependencies will have to be removed when we include the access service here